solidity/dashboard/src/reducers/staking.js

```diff
@@ -150,7 +156,7 @@ const topUpInitiated = (topUps, { operator, topUp }) => {
       {
         operatorAddress: operator,
         availableTopUpAmount: topUp,
-        createdAt: moment.unix(),
+        createdAt: moment().unix(),
       },
       ...topUps,
```

Comment:
The commit-top up button is still disabled (until we refresh) because here we add a topUp (lines 156-159) and we did not define `readyToBeCommitted` variable so it's undefined.

Reply:
So we have to think of a way to update this newly created top-up with `readyToBeCommited = true` variable, without refreshing the page but i don't think it's possible without fetching the topUps from BE again 🤔 Any ideas?

Reply:
The top-up will be updated in `notifyTopUpReadyToBeCommitted` saga. Before displaying notification we dispatch `staking/top_ups_ready_to_be_committed` action which should update top-ups that are ready to be commited.